README.md

```diff
@@ -94,14 +94,89 @@ The output format for Syft is configurable as well:
 syft packages <image> -o <format>
 ```
 
-Where the `format`s available are:
+Where the `formats` available are:
 - `json`: Use this to get as much information out of Syft as possible!
 - `text`: A row-oriented, human-and-machine-friendly output.
 - `cyclonedx`: A XML report conforming to the [CycloneDX 1.2 specification](https://cyclonedx.org/specification/overview/).
 - `spdx`: A tag-value formatted report conforming to the [SPDX 2.2 specification](https://spdx.github.io/spdx-spec/).
 - `spdx-json`: A JSON report conforming to the [SPDX 2.2 JSON Schema](https://github.com/spdx/spdx-spec/blob/v2.2/schemas/spdx-schema.json).
 - `table`: A columnar summary (default).
 
+## Private Registry Authentication
+
+Syft uses the [authn](https://github.com/google/go-containerregistry/tree/main/pkg/authn) package to solve for private registry access. 
```

Comment:
The use of authn is more of an implementation detail that doesn't need to be highlighted specifically. The more important point to bring forward is that syft will JustWork™ in absence of a container runtime. Something like:

```suggestion
When a container runtime is not present Syft can still utilize credentials configured in common credential configuration sources (such as `~/.docker/config.json`) and pull images from private registries. 
```

Reply:
It's true that it is an implementation detail, but their readme was invaluable in helping me understand the auth flow when I was trying to identify how everything stacked together. Not including the link/info for future users to discover felt like we would be missing a key piece of well-written docs. I can update the phrasing.

Reply:
Updated phrasing. Kept follow up link removing reference to `authn` and instead referenced `go-containerregistry`